### ISSUE TITLE
Error in database generation 2

### ISSUE THREAD

**Issue by minhtrung1997** [liaoherui/VirStrain#10]
[MAFFT.clustal.fasta.txt](https://github.com/liaoherui/VirStrain/files/10265643/MAFFT.clustal.fasta.txt)

The error is:
[error.txt](https://github.com/liaoherui/VirStrain/files/10265659/error.txt)

The command run is:
```
#!/bin/bash -euo pipefail
virstrain_build -i MAFFT.clustal.fasta -d VIRSTRAIN_DB -s 1
echo -e "VIRSTRAIN version 1.12" > virstrain.version.txt
```

**Comment by liaoherui** [comment]
Hi, this is a bug for bioconda version of VirStrain, will fix it asap.

You can try the github version, which has been tested by me and it works fine.

**Comment by liaoherui** [comment]
Hi, I have fixed this bug in v1.13. The package has been updated in PyPI, and the Bioconda version v1.13 will be updated automatically later.

Thanks for using VirStrain. Any other problems or suggestions, feel free to let me know.
